additional-data/scripts/update-timeline-stats.js

```diff
@@ -0,0 +1,204 @@
+/**
```

Comment:
Similar to my comment about keeping the `additional-data` folder for manually maintained augmented web-features data, could you move the script to somewhere else? At the root of the repo would be fine I think.